altapay/api.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+import os
+import platform
+import ssl
+from xml.etree import ElementTree
```

Comment:
I don't want to depend on another external library for a third party package, and the documents are typically rather small, so the gain is insignificant.